mava/configs/default.yaml

```diff
@@ -39,7 +41,12 @@ absolute_metric: True # Whether the absolute metric should be computed. For more
   # on the absolute metric please see: https://arxiv.org/abs/2209.10485
 
 # --- Logging options ---
-use_tf: True
+use_tf: False
 use_sacred: True
+split_json_by_seed: True # Split JSON logs by seed after the full experiment has run.
```

Comment:
I thought about making it optional because postprocessing can take some time if you record for many evaluations and if someone don't care about splitting can skip that time (we also made a warning that will shows up if this is false so people be aware of it)